cli/types/types.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 Globo.com authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package types
```

Comment:
But why wasn't this file here before?

Reply:
It was being imported from the `client` component [here](https://github.com/globocom/huskyCI/pull/481/files#diff-4826362d61b88bc316da9d728b970b7aR8). 😅As we are deciding to separate dependencies from each component, I am including this file inside the `cli` component folder as well.